install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -35,6 +35,7 @@ items:
       release: {{ .Release.Name }}
       heritage: {{ .Release.Service }}
   spec:
+    type: {{ .Values.service.uiType }}
```

Comment:
Why adding this type? I think @costinm 's proposal is using ingressgateway as the central entry point to access all services in istio chart.